tests/zfs-tests/tests/functional/cli_root/zpool_trim/cleanup.ksh

```diff
@@ -0,0 +1,36 @@
+#!/bin/ksh -p
+#
+# CDDL HEADER START
+#
+# The contents of this file are subject to the terms of the
+# Common Development and Distribution License (the "License").
+# You may not use this file except in compliance with the License.
+#
+# You can obtain a copy of the license at usr/src/OPENSOLARIS.LICENSE
+# or http://www.opensolaris.org/os/licensing.
```

Comment:
opensolaris.org is no more. Please refer to http://opensource.org/licenses/CDDL-1.0